### PR TITLE
removed preview and made download, added validation code, language changes

### PR DESCRIPTION
**Pull request by miabonardi**

### CODE REVIEW

File: docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ question: |
   % if email_success:
   Your form is on its way.
   % else:
-  Something went wrong delivering your form
+  Something went wrong delivering your form.
```
Comment (BryceStevenWilley) [inline docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml]: Do we prefer periods on headers / page title text that is larger than others? I'd lean towards not having periods, but not sure what the rest of the interviews have been doing, and can't find anything about it in the Plain language docs.

File: docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml [hunk 2]
```diff
@@ -136,39 +136,41 @@ question: |
   Final Review With Cover Page
 subquestion: |
   
-  Below is your ${comma_and_list(al_download_titles)} document with the
-  cover page that we will deliver to ${courts[0]}.
+  Below is your document with the 
+  cover page that will be delivered to the ${courts[0]}.
```
Comment (BryceStevenWilley) [inline docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml]: I'm okay with removing the titles, but we prefer [active voice](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/content_and_plain_language_guide#write-in-active-voice-not-passive-voice), not passive voice, right?

File: docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml [hunk 3]
```diff
@@ -136,39 +136,41 @@ question: |
...
   Click "Back" if you need to make any changes.  
-  Click "Send to court" to deliver it.
+  Click "Send to the ${courts[0]}" to deliver it.
   
   % if users[0].email:
   **By sending this form to the court, you agree that you
   will get notices from the court by email.** If you do not want to receive
   notices by email, click "Back", download your form, and send it to court
-  by mail or by dropping it off.
+  by mail or drop it off.
```
Comment (BryceStevenWilley) [inline docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml]: I'm not sure it flows here, there are a lot of prepositions? What about:

```
download your form, and mail it to the court or drop it off.
```

File: setup.py [hunk 4]
```diff
@@ -53,7 +53,7 @@ def find_package_data(where='.', package='', exclude=standard_exclude, exclude_d
       url='https://docassemble.org',
       packages=find_packages(),
       namespace_packages=['docassemble'],
-      install_requires=['docassemble.MAVirtualCourt'],
+      install_requires=['docassemble.MAVirtualCourt>=1.0.20'],
```
Comment (BryceStevenWilley) [inline setup.py:56]: I think this is fine, keeping other packages up to date is always a good idea.

**Review by BryceStevenWilley — COMMENTED** [review]
There are a few nit picks, but overall looks good! You don't have to change everything I suggested, but let me know if you have reasons not to, and I'll merge.

File: docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml (hunk 1 above)
Comment (miabonardi) [reply docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml]: Changed back in new commit. You're definitely right! I did not realize it was in the question.

File: docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml (hunk 2 above)
Comment (miabonardi) [reply docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml]: Changed in new commit. Should definitely be active voice.

File: docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml (hunk 3 above)
Comment (miabonardi) [reply docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml]: Changed in new commit.

**Review by BryceStevenWilley — APPROVED** [review]

**Pull request merged**